doc/docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,105 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'solid_lints',
+  tagline: 'solid_lints are cool',
+  url: 'https://solid-software.github.io/',
+  baseUrl: '/solid_lints/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.png',
+  organizationName: 'solid_software', // Usually your GitHub org/user name.
+  projectName: 'solid_lints', // Usually your repo name.
+  deploymentBranch: 'web-docs-deployment',
```

Comment:
before merge, we should be sure that this branch is exists and it is selected as default branch for github pages in settings.